Team 13: Reimplemented on-screen result messages (removed alerts)

SadiaK-art opened the pull request.

BlueJ1 committed:
feat: first commit

BlueJ1 committed:
feat: new version

mast1554 committed:
made more texts dynamic

Roubleman committed:
Streets added

BlueJ1 committed:
feat: first version

BlueJ1 committed:
feat: new first version

BlueJ1 committed:
Merge pull request #3 from BlueJ1/team8/feat/settings

Team8/feat/settings and (alternative) backend

BlueJ1 committed:
feat: improved end-screen; added word details page

BlueJ1 committed:
feat: update end-screen and words-detail to load game results from Team 8 store

mast1554 committed:
Merge branch 'main' into team12/language-toggle

BlueJ1 committed:
feat: implement welcome page layout and navigation

BlueJ1 committed:
refactor: update welcome page styles and navigation links

BlueJ1 committed:
fix: fix issues after rebase

BlueJ1 committed:
refactor: remove unnecessary strict mode declaration from index.js

BlueJ1 committed:
refactor: reorder import statements in index.js

BlueJ1 committed:
style: update font-family in index.css for improved typography

BlueJ1 committed:
feat: enhance image handling in words detail; remove exit button from words detail screen

Darkfrobozz committed:
Putting settings in its own module.

Darkfrobozz committed:
Merge pull request #4 from BlueJ1/team8/feat/welcome-page

team8/feat/welcome-page

Roubleman committed:
Updated streets json to match map better

BlueJ1 committed:
refactor: update category display and remove exit functionality from words detail

tbimbato committed:
added placeholder for various functions in index.js

tbimbato committed:
update index.js

BlueJ1 committed:
feat: update index.html and index.js for improved game navigation and database initialization

Darkfrobozz committed:
added alternative backend and limited interface

Darkfrobozz committed:
fixed type error

Darkfrobozz committed:
small type typo fixed

Darkfrobozz committed:
Merge pull request #5 from BlueJ1/team8/fix/bug_fixes

Team8/fix/bug fixes

Darkfrobozz committed:
fixed type error

Darkfrobozz committed:
Merge pull request #6 from uu-semp/main

Updating fork

mast1554 committed:
added dynamic text for title and description + bugfix

mast1554 committed:
added dynamic texts for chapter filters

mast1554 committed:
Merge branch 'main' into team12/language-toggle

tbimbato committed:
Add clothes PNG, initial files for description generator and project notes.

- Created descr_generator.js for future functionality
- Added notes.txt to outline project decisions and direction

tbimbato committed:
Implement Swedish clothing description generator and add CSV that links png clothes to category and description

SimonPislar committed:
Implement Swedish clothing description generator and add CSV index for png files (#81)

tbimbato committed:
Refactor index.jsm update notes.txt with current function status

SaadiehV committed:
simple functionality of selection process

simple functionality of selection process has been created without any styiling.

BlueJ1 committed:
feat: enhance button styles and remove exit functionality; add back button to words detail

SaadiehV committed:
main game mostly done

Main game functionality
code is not properly commented
might be some errors in naming conventions etc

tzeenht committed:
initial css and html for game and menu files

Darkfrobozz committed:
Merge pull request #9 from uu-semp/main

Audio file fixes are merged

Darkfrobozz committed:
adding Viktor's maingame

SaadiehV committed:
Merge branch 'team8/feat/maingame' of https://github.com/BlueJ1/swedish-learning-app-2025-team8 into team8/feat/maingame

Saimpan committed:
Began implementing loadClothes

Saimpan committed:
Implement loadClothes and helper functions

Saimpan committed:
Remove comment

Ramizoro committed:
adding the first draft of main game. 1 jtml file, 1 css file and 1 js file. it satisfies requirement 8,9 and 13. adding 2 audio files for correct/incorrect answer temporarily.

Darkfrobozz committed:
added images and created basic selection screen (checkboxes need still to be fixed)

Darkfrobozz committed:
removed some dead code and restructured css

SaadiehV committed:
Merge branch 'team8/feat/maingame' of https://github.com/BlueJ1/swedish-learning-app-2025-team8 into team8/feat/maingame

Saimpan committed:
Add feature to loadClothes where it can insert loaded imgs to the html page.

BlueJ1 committed:
fix: improve image handling in words detail; add warning for missing image URLs

mast1554 committed:
Merge branch 'main' into team12/language-toggle

Roubleman committed:
Updated streets.json with coords

poojanagalikar committed:
create team vocab by mapping street and vocabulary json

poojanagalikar committed:
Merge branch 'team16/backendbranch' of https://github.com/uu-semp/swedish-learning-app-2025 into team16/backendbranch

BlueJ1 committed:
refactor: shorten button text

Darkfrobozz committed:
added reset

Darkfrobozz committed:
Merge pull request #12 from BlueJ1/team8/feat/reset

added reset

Darkfrobozz committed:
Applied reset to select_screen and fixed differences in look as a result

SaadiehV committed:
Merge branch 'team8/feat/maingame' of https://github.com/BlueJ1/swedish-learning-app-2025-team8 into team8/feat/maingame

Ramizoro committed:
updating the main game. Changed color scheme. changed UI. Removed main_game_component folder.

Ramizoro committed:
Merge branch 'team8/feat/maingame' of https://github.com/BlueJ1/swedish-learning-app-2025-team8 into team8/feat/maingame

Darkfrobozz committed:
Merge branch 'main' into team8/feat/maingame

Darkfrobozz committed:
added reset.css

Darkfrobozz committed:
Merge pull request #10 from BlueJ1/team8/feat/select_screen

Team8/feat/select screen

Darkfrobozz committed:
merged with main

Darkfrobozz committed:
Merge pull request #8 from BlueJ1/team8/feat/end-screen

team8 (feat): end screen

Darkfrobozz committed:
merged with main

Darkfrobozz committed:
Merge pull request #13 from BlueJ1/team8/feat/maingame

Team8/feat/maingame

Darkfrobozz committed:
Integrating all components to allow for fully playable game

Darkfrobozz committed:
Merge pull request #14 from BlueJ1/team8/feat/integrate

Integrating all components to allow for fully playable game

Darkfrobozz committed:
Merge pull request #15 from uu-semp/main

Merging from upstream

Viv9000 committed:
fix: uu logo removed from header

mast1554 committed:
remake of translations json and the languagle toggle function

mast1554 committed:
fixed initialization

Roubleman committed:
Updated streetsjson with x, y, width, height, position

Roubleman committed:
Merge branch 'team16/backendbranch' of https://github.com/uu-semp/swedish-learning-app-2025 into backendbranch

poojanagalikar committed:
changed the fetch external value to true

poojanagalikar committed:
team vocab data with coordinates

Saimpan committed:
Update htmlImgObject to take id instead of description

Darkfrobozz committed:
added some more tests and fixed some bugs

Darkfrobozz committed:
added end to end tests for selection component

Darkfrobozz committed:
renamed and set correct defaults

Darkfrobozz committed:
corrected a spelling mistake

BlueJ1 committed:
fix: Close h2 correctly

Saimpan committed:
Complete loadClothes and with ability to move clothing items included

Saimpan committed:
Merge remote-tracking branch 'origin/team14_backend_loadClothes' into team14_backend_loadClothes

# Conflicts:
#	team14/index.js

Saimpan committed:
For testing

Saimpan committed:
Merge remote-tracking branch 'origin/team14_backend_loadClothes' into team14_backend_loadClothes

mast1554 committed:
added back- and filter-buttons totranslations

mast1554 committed:
Merge branch 'main' into team12/language-toggle

Saimpan committed:
Add comments to the code

Saimpan committed:
Add TODOs and remove old function placeholders

Darkfrobozz committed:
Merge branch 'refactor/backend' into team8/feat/backend_compliance
Adding alternative_backend

Darkfrobozz committed:
added loading of main menu local storage script

Darkfrobozz committed:
complying with the main menu's backend

Darkfrobozz committed:
refactoring to make overlook clearer and using new limited backend approach

Darkfrobozz committed:
fixed read.js bug

Darkfrobozz committed:
renamed testing in selectiontest.html

Darkfrobozz committed:
fixed test concurrency errors and removed internal alternative_backend

Darkfrobozz committed:
await now only executes at proper location

Ramizoro committed:
SCRUM-146-Add Keyboard Accessibility to the main game

Ramizoro committed:
Merge branch 'main' into team08/feat/keyboardAcessToGame

klintskar committed:
Renamed clothing items for jackets to include the jacket in the name

tzeenht committed:
added content for game.html and game.css

tbimbato committed:
added area column in cothing_database.csv,
(swedish descr generator still on WIP

klintskar committed:
Added scarf png and to csv file, also updated csv jacket name

klintskar committed:
Merge branch 'team14_backend' of https://github.com/uu-semp/swedish-learning-app-2025 into team14_backend

Roubleman committed:
setting up levels function and local storage

tbimbato committed:
description generator funcion is now not hardlinked to png files paths

Saimpan committed:
Merge branch 'team14_backend_loadClothes' into team14_backend

# Conflicts:
#	team14/index.js

Saimpan committed:
Fixed errors as a result of merging

tbimbato committed:
Enhance clothing category normalization and inference in SwedishClothingDescriptionGenerator (since csv changed columns)

tbimbato committed:
Merge branch 'team14_backend' into team14_backend_loadClothes

tbimbato committed:
Team14 backend load clothes (#146)

Add load clothes function along with helper functions. Request merging
this feature branch to the main team14_backend branch.

tbimbato committed:
modified the description generator function in order to use right img class

SimonPislar committed:
Team14 backend merge funcs (#147)

first merge of backend js file into sub-main

tzeenht committed:
added more clothing categories

Roubleman committed:
Merge branch 'main' of https://github.com/uu-semp/swedish-learning-app-2025 into backendbranch

Saimpan committed:
Removed testing code

Roubleman committed:
Removed last date played and time spent from gameProgress

tzeenht committed:
Merge branch 'team14_frontend' into team14

Saimpan committed:
Adapt loadClothes function to work with frontend code

Saimpan committed:
Add data-accept tags to clothing boxes. Import js file.

Saimpan committed:
Add data-accept tags to clothing boxes. Import js file.

Saimpan committed:
Add styling for the clothing menu. Moved the hat a bit further up.

Saimpan committed:
Add ids to clothing boxes in order to identify the easily

Saimpan committed:
Modified checkClothes function to work with new implementation

klintskar committed:
Added all assets in the global assets folder and have a test loading all assets from our local team14 folder.

tzeenht committed:
confirm button

tzeenht committed:
Merge remote-tracking branch 'origin/team14' into team14_integration

BlueJ1 committed:
refactor: remove completion progress bar (#18)

mast1554 committed:
removed flags and about-btn, fixed size and position of header buttons

mast1554 committed:
added documentation and removed fixed position (since fixed size)

mast1554 committed:
Merge branch 'main' into team12/language-toggle

Darkfrobozz committed:
Merge branch 'main' into team8/feat/backend_compliance

harikrishnan-vinod committed:
backend and frontend connected

Darkfrobozz committed:
renamed internal alternative backend to backend_interface

Darkfrobozz committed:
Merge pull request #17 from BlueJ1/team8/feat/backend_compliance

Team8/feat/backend compliance

Added backend compliance.
Our store now goes through save.js.
Moreover, our backend implementation is an addition in the backend.

Darkfrobozz committed:
Merge pull request #19 from uu-semp/main

Merging from main

Darkfrobozz committed:
Updating save with interface to stats

Darkfrobozz committed:
added progress update function and basic win and progress counting mechanisms

Roubleman committed:
Corrected grammar

Roubleman committed:
Merge branch 'team16/backendbranch' of https://github.com/uu-semp/swedish-learning-app-2025 into backendbranch

Darkfrobozz committed:
Added progress update call to finish_game

Adar10 committed:
Team07 (#81): Changed description of button

Adar10 committed:
Team07 (#81): When clicking on image only the correct answer and the clicked answer will be highlighted. Also added text to reveal the correct answer

poojanagalikar committed:
changes in json files

Darkfrobozz committed:
merged in upstream

Darkfrobozz committed:
removed old alternative_backend structure and updated interface to reflect this update

Darkfrobozz committed:
Merge pull request #22 from BlueJ1/upstream_merge

Upstream merge

Darkfrobozz committed:
Merge pull request #20 from BlueJ1/team8/feat/stats_compliance

Team8/feat/stats compliance

mast1554 committed:
Merge branch 'main' into team12/language-toggle

Adar10 committed:
Team07 (#81): Added logic for loading categories before allowing selection

VictorGB01 committed:
Add missed words replay functionality

mast1554 committed:
updated structure of games.json and added words to menu_translations

gastontornquist committed:
Multi-filter-chaptering

Filter button now lets you select several chapters
Filter feature displays games containing any of the selected chapters

gustavmolin committed:
add licensing for assets

Roubleman committed:
Updated team16 vocab

Roubleman committed:
Merge branch 'team16/backendbranch' of https://github.com/uu-semp/swedish-learning-app-2025 into backendbranch

Roubleman committed:
team16 vocab json corrected

Saimpan committed:
Add saving of wins

tzeenht committed:
Merge branch 'team14_integration' of https://github.com/uu-semp/swedish-learning-app-2025 into team14_integration

klintskar committed:
Removed old testing file

klintskar committed:
Renamed menu to index

klintskar committed:
Fixed so main menu is index.html

SimonPislar committed:
Team14 integration (#192)

Moving team14_integration branch to team14. Finished game

klintskar committed:
Removed unused files and assets

Saimpan committed:
Remove unused clothing boxes

Saimpan committed:
Remove function call to checkClothes used for testing

ivje9902 committed:
Team07/Add missed words replay functionality (#189)

Bokaka09 committed:
Team7/81 fix feedback logic (#185)

Fixed, feedback logic so that only the clicked answer and correct answer
are highlighted.
Added a loading feature that waits for data from backend to be loaded
before category selection.

xFrednet committed:
Merge remote-tracking branch 'origin/main' into team16/backendbranch

harikrishnan-vinod committed:
team16/fixed level progression issue

harikrishnan-vinod committed:
Merge branch 'team16/backendbranch' of https://github.com/uu-semp/swedish-learning-app-2025 into team16/backendbranch

Viv9000 committed:
Feat: game card images

Ihrefjord committed:
Team16/backendbranch (#101)

Mapped both the street and vocabulary JSON to create teams vocab JSON
file.

Fixed game progression bug

Fixed vocabulary mismatches

Viv9000 committed:
feat: desc and titles

Viv9000 committed:
fix: forgot to remove logo png

Viv9000 committed:
feat: follow style guide red color

Viv9000 committed:
bugfix: desc no longer has accidental "}" at the end

SebbeBlom committed:
Team12/titles desc image and chapters (#197)

this PR updates titles, description, logo and supported chapter tags all
games

and updates red color to follow our own style guide

LudvigMagdeburg-Official committed:
Team13/main (#198)

### Data 
* Added completion tracking
* Implemented points screen
* Implemented end screen
* Tweaked progress bar

TODO:
- Make end screen prettier
- Remove alerts
- Center start page and end screen

SadiaK-art committed:
Reimplemented on-screen result messages (removed alerts)

Gytis0 committed:
Pseudo-random algorithm to fetch sets which contain mostly new unlearned words

martineken committed:
SCRUM-198/SCRUM-199: Fixed so styling and design now matches main menu

SebbeBlom committed:
team12/Multi-filter-chaptering (#191)

Filter button now lets you select several chapters
Filter feature displays games containing any of the selected chapters

theocarlsson committed:
Team03: Styling and design now matches main menu (#201)

Tickets included:

SCRUM-198/SCRUM-199: Fixed styling to match main menu

EmilEdin committed:
Fixed progress bar visibility, lvl1-3 bug fixes.

mast1554 committed:
fixed translations for game grid

mast1554 committed:
Merge branch 'main' into team12/language-toggle

mast1554 committed:
fix merge

alliliu committed:
Team10/Fixed progress bar visibility, lvl1-3 bug fixes. (#202)

Fixed progress bar visibility so that It can be seen from main menu when
you click on the game. Also fixed bugs in level 1, level 2 and level 3
and integrated advance to next level pages.

simongebreyohannes committed:
level3 done, i think

SebbeBlom committed:
Team12/remove UU logo from header (#109)

Removes the UU logo from the header

Resolves issue #40

abandon1232 committed:
Final game

abandon1232 committed:
Final game version (#204)

tbimbato committed:
Team 14 Game (#193)

We in team 14 have created a functional game and want to merge our code
to main for further testing.

HectorBergman commented on team13/game.js:
Don't change this

Ramizoro committed:
updated paths for audio and images. Also updated rounds to 10.

HectorBergman commented on team13/game.js:
Delete this, it has been moved to houses.js

HectorBergman commented on team13/game.js:
Uncomment this

HectorBergman reviewed (changes requested):
Submitted several requested changes that need to be addressed. Also the alert for finishing a game still remains.

Also, add back point regression from answering incorrectly. As of right now it just waits until you've clicked the correct house.

Darkfrobozz committed:
fixed urls

mast1554 committed:
fixed translations for filter tags

mast1554 committed:
Merge branch 'main' into team12/language-toggle

Darkfrobozz committed:
added settings and keyboard controls for selection and index and fixed bug in write

Ramizoro committed:
synched global volumn and sound effect to main game volumn and sound effect

SebbeBlom committed:
Team12/language toggle (#77)

Added functionality for changing the language on the main menu between
Swedish and English. The text now gets fetched from one of two .json
files (one for Swedish and one for English). Clicking one of the
language buttons rerenders all elements with text to now display the
correct language.

TODO: code neeeds to be updated when PR #56 is merged into main

Ramizoro committed:
code merged and tested. also fixed minor bugs

Ramizoro committed:
synched global volumn and sound effect to main game volumn and sound effect

Darkfrobozz committed:
fixed settings styling

Darkfrobozz committed:
merged in main and solved conflicts

Darkfrobozz committed:
Merge pull request #23 from BlueJ1/team08/feat/keyboardAcessToGame

Team08/feat/keyboard acess to game

Needs changes but we will merge to try it out

Miro-Ali-Akbar committed:
Fixes a smal logic error

simongebreyohannes committed:
Team02 (#207)

Darkfrobozz committed:
updated main game to use settings and updated styling of settings buttons

simongebreyohannes committed:
.

simongebreyohannes committed:
Fixes a smal logic error (#206)

Darkfrobozz committed:
added animation for pressing

Miro-Ali-Akbar committed:
. (#208)

SadiaK-art committed:
Update team13/game.js

Co-authored-by: HectorBergman <113025156+HectorBergman@users.noreply.github.com>

Darkfrobozz committed:
updated to not use db on start page

SadiaK-art committed:
Update team13/game.js

Co-authored-by: HectorBergman <113025156+HectorBergman@users.noreply.github.com>

SadiaK-art committed:
Update team13/game.js

Co-authored-by: HectorBergman <113025156+HectorBergman@users.noreply.github.com>

SadiaK-art committed:
Update team13/game.js

Co-authored-by: HectorBergman <113025156+HectorBergman@users.noreply.github.com>

SadiaK-art committed:
Update team13/game.js

Co-authored-by: HectorBergman <113025156+HectorBergman@users.noreply.github.com>

maxeby committed:
Score

changed how the score is displayed

Darkfrobozz committed:
only count first

Darkfrobozz committed:
fixed audio bug

lkoppe committed:
Score (#209)

changed how the score is displayed

antonsordal committed:
Team15/license/#161#162

Functionality for button displaying correct licenses for the images

Darkfrobozz committed:
fixed saving error as they want percentage form rather than decimal

antonsordal committed:
Team15/license/#161#162 (#211)

Functionality for in-game button displaying correct licenses for all
images, currently only visible in Level 1.

Darkfrobozz committed:
added a tutorial with some information about how the game works and some information about keybindings

Darkfrobozz committed:
added persistent storage notice

Darkfrobozz committed:
made notice less vague

olivialundbergh committed:
added readme file

Darkfrobozz committed:
some components were still saving directly to local storage

olivialundbergh committed:
Added readme file (#212)

Added the README file, created by Abdur Rheman.

Darkfrobozz committed:
Merge pull request #24 from BlueJ1/team08/feat/keyboardAcessToGame

Team08/feat/keyboard acess to game

Darkfrobozz committed:
Merge pull request #25 from uu-semp/main

Merging main into our fork

Darkfrobozz committed:
added copyright information to main game

Darkfrobozz committed:
fixed persistent notice to not blink when loading in by setting default to be none

Darkfrobozz committed:
added styling for copyright button

Darkfrobozz committed:
Merge pull request #26 from BlueJ1/feat/copyright

Feat/copyright

SadiaK-art committed:
Fixed typos, game logic, connected game.html

- Fixed typos in the code
- The game logic is back to normal now
- Removed the last alert and added the text to the end screen instead

LoppanSune committed:
Team08 - Merging game (#213)

This pull request contains team08's version of the "say what?" game.

HectorBergman committed:
Progressmin to 0

HectorBergman committed:
small tweaks

HectorBergman committed:
Merge branch 'main' into team13/frontend/alert-removal-v2

HectorBergman committed:
Prevented spam-clicking
